### ISSUE TITLE
should παρήγγελλεν in Luke 8.29 be imperfect rather than aorist?

### ISSUE THREAD

**Issue by jtauber** [morphgnt/sblgnt#61]

**Comment by seumasjeltzz** [comment]
I think you should tag it as imperfect because the form is imperfect, isn't it? You can't tag based on textual variations

**Comment by jtauber** [comment]
@seumasjeltzz Yeah, I just wanted a second opinion in case I was missing something. Finding this was an unexpected consequence of writing my next morphology tour blog post :-)

Oddly:

1) Logos tags it aorist (even for the παρήγγελλεν form) which makes me wonder if they just transferred an analysis from a different text (like I did back in 2010).

2) Morpheus gives both an aorist and an imperfect analysis as possibilities.
